Main.java

```diff
@@ -0,0 +1,186 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void Get_Median(String []arr) {
+        int TempArr[]=new int[arr.length];
+
+        if(!Character.isDigit(arr[0].charAt(0))) {
+            System.out.println("Median : Sorry this function run with array of numbers only !!");
+            return;
+        }
+
+        //Take Copy from the original array
+        for(int i=0;i<arr.length;i++) {
+            TempArr[i]=Integer.parseInt(arr[i]);
+        }
+
+        //Sort Array (Bubble Sort) (N^2)
+        for(int i=0;i<TempArr.length;i++) {
+            for(int j=0;j<TempArr.length;j++) {
+                //Swap
+                if(TempArr[i]>TempArr[j]) {
+                    int temp=TempArr[i];
+                    TempArr[i]=TempArr[j];
+                    TempArr[j]=temp;
+                }
+            }
+        }
+
+        //Get Median
+        double Median=0.0;
+
+        //if size of array is even
+        if(TempArr.length%2==0) {
+            int index1=(TempArr.length/2)-1;
+            int index2=TempArr.length/2;
+            Median=(TempArr[index1]+TempArr[index2])/2.0;
+        }
+        //if size of array is odd
+        else {
+            int index=(TempArr.length/2);
+            Median=TempArr[index];
+        }
+
+        //Output
+        System.out.println("Median = "+Median);
+    }
+    //ZERO IF LESS THAN ZERO
+    public static void zeroIfLessThanZero(String[] arr) {
+        //copying the original array into temporary one
+        int temp[] = new int[arr.length];
+        for (int i = 0; i < arr.length; i++) {
+            temp[i] = Integer.parseInt(arr[i]);
```

Comment:
the Integer parser throws NumberFormatException if it doesn't find integer characters.
I suggest something like this:
try{
     temp[i] = Integer.parseInt(arr[i]);
}catch(NumberFormatException e){
     //a message or something to handle the exception
}

Reply:
i recommend @BassemReda suggestion to avoid throw errors